src/renderer/components/data-settings/data-settings.js

```diff
@@ -129,6 +114,25 @@ export default Vue.extend({
         })
         return
       }
+      response.filePaths.forEach(file => {
+        if (file.endsWith('.csv')) {
```

Comment:
automatically determine subscription type instead of having the user specify it when importing